### PR TITLE
Clean up code for CREATE/CALL

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

**Comment by axic** [comment]
@holgerd77 @jwasinger please review this too

**Comment by holgerd77** [comment]
What is going on here with the 1 failed test (``CallToNameRegistratorTooMuchMemory2``), is this intended/expected?

**Comment by axic** [comment]
Need to be figured out why, see no reason for it yet.

**Comment by jwasinger** [comment]
Not able to run this:
```
/home/jwasinger/projects/axic/ethereumjs-vm/tests/tester:85
  testing.runTests(runner, tests, tape, skip, cb)
          ^

TypeError: testing.runTests is not a function
    at runTests (/home/jwasinger/projects/axic/ethereumjs-vm/tests/tester:85:11)
    at Object.<anonymous> (/home/jwasinger/projects/axic/ethereumjs-vm/tests/tester:37:3)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Function.Module.runMain (module.js:605:10)
    at startup (bootstrap_node.js:158:16)
    at bootstrap_node.js:575:3
```

File: lib/opFns.js [hunk 1]
```diff
@@ -581,7 +588,6 @@ module.exports = {
       })
 
       try {
-        checkCallMemCost(runState, options, localOpts)
```
Comment (jwasinger) [inline lib/opFns.js]: This is where the issue is.  Add this line back in to make the test pass.

Comment (axic) [inline lib/opFns.js]: Yeah but it makes no sense as the `localOpts` are not changing at that point. Does the test expect gas deducted twice? The reason must be understood.

Comment (axic) [inline lib/opFns.js]: Derp. It is never called in that function before. Note to self: shouldn't do these PRs late in the evening.

**Pull request merged**